interface/weno/confirm.php

```diff
@@ -0,0 +1,258 @@
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+$fake_register_globals=false;
+$sanitize_all_escapes=true;	
+include_once('../globals.php');
+include_once('transmitDataClass.php');
+include_once('$srcdir/patient.inc');
+
+$date = date("Y-m-d");
+$pid = $GLOBALS['pid'];
+$uid = $_SESSION['authUserID'];          //username of the person for this session
+
+$tData = new transmitData();
+
+$send = $tData->getDrugList($pid, $date);
+$provider = $tData->getProviderFacility($uid);
+$patientPharmacy = $tData->patientPharmacyInfo($pid);
+$mailOrder = $tData->mailOrderPharmacy();
+
+
+?>
+
+<html>
+<head>
+<?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-ui-1-10-4/themes/dot-luv/jquery-ui.min.css" type="text/css" />
+
+<script type="text/javascript" charset="utf-8">
+function validate(){
+	var pharmacy = document.getElementById('pharm');
+	if(text.value.length == 0){
+		alert('<?php print xlt("Must select a pharmacy first"); ?>');
+		return false;
+	}
+}
+
+
+</script>
+<style>
+footer {
+    padding: 0.5em;
+    font-size: 0.5em;
+
+    clear: left;
+    text-align: center;
+    top: 200px;
+}
+</style>
+</head>
+
+<body class="body_top">
+<h1><?php print xlt("Prescription Transmit Review"); ?></h1>
+<table>
+<th width="200px"><?php print xlt("Drug"); ?></th>
+<th width="100px"><?php print xlt("Quantity"); ?></th>
+
+<?php
+//List drugs to be sent 
+
+  $drug = array(); //list of records that need to updated with pharmacy information
+while($list = sqlFetchArray($send)){
+	print "<tr align='center'><td>". xlt(text($list['drug'])) . " </td><td> " . xlt(text($list['quantity'])) . "</td></tr>";
+	$drug[] = $list['id'];
+}
+
+
+?>
+</table>
+<?php if(empty($drug)){
+	echo "<br> <strong> <font color='red'>".xlt("No prescriptions selected"). "</strong></font>";
+	exit;
+}
+?>
+<div id="fields">
+<h3><?php echo xlt("Select Pharmacy"); ?></h3>
+	    Patient Default <br>
+	    <input type = 'radio' name = "pharmacy" id = 'patientPharmacy' value="<?php print attr($patientPharmacy['pharmacy_id']) ?>" checked="checked">
+	    <?php if(!$patientPharmacy['name']){
+                   print "<b>".xlt("Please set pharmacy in patient\'s chart!")."</b><br> <br>";
+               }else{
+	    	      print xlt(text($patientPharmacy['name'])); 
```

Comment:
I think I was being proactive and trying to get ahead of the review. So, anything that is going to be printed on the screen should be translated?